README.md

```diff
@@ -1 +1,64 @@
-# JavaScipt-capstone-2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Preview
+<p align="center">Desktop view</p>
+<img src="src/assets/images/Screenshot from 2022-04-24 14-54-51.png">
+<img src="src/assets/images/Screenshot from 2022-04-24 14-55-48.png">
+<img src="src/assets/images/Screenshot from 2022-04-24 14-56-26.png">
+
+<h1 align="center"> Tasty Eats </h1>
+
+> The website allows users to browse a list of meals, like a meal, view and add comments to a meal.
+
+
+## Built With
+
+- **HTML** and **CSS** and **Javascript**
+
+## DEMO
+
+[LIVE DEMO LINK](https://maenkhraisha.github.io/JavaScript-capstone-2/)
```

Comment:
- [ ] **Optional:** Your live link is broken, it does not display the dishes, kindly fix this.